src/pages/Background/index.js

```diff
@@ -1583,14 +1583,14 @@ const checkAvailableMemory = (sendResponse) => {
 };
 
 // Add this function to handle sign out from GoDAM
-const handleSignOutGoDAM = async () => {
+const handleSignOutGoDAM = async (sendResponse) => {
```

Comment:
This file is already too long. Do you mind refactoring this function out into a different function in the modules dir.